src/components/BuildConfigCreateEditPage/BuildConfigCreateEditPage.tsx

```diff
@@ -932,15 +969,49 @@ export const BuildConfigCreateEditPage = ({ isEditPage = false }: IBuildConfigCr
         </ContentBox>
       )}
 
-      <ActionGroup>
-        <Button
-          variant="primary"
-          isDisabled={isSubmitDisabled || (selectedProduct && !selectedProductVersion) || serviceContainerScmRepositories.loading}
-          onClick={handleSubmit(isEditPage ? submitEdit : submitCreate)}
-        >
-          {isEditPage ? ButtonTitles.update : ButtonTitles.create} {EntityTitles.buildConfig}
-        </Button>
-      </ActionGroup>
+      <Form>
```

Comment:
Form for just button does not seem semantically right

Reply:
ActionGroup is assumed to be inside form, otherwise stylesheets are broken, the thing is that the whole page you created previously is kinda specific as it's splitted into several forms. Once NCL-8445 is started, probably whole ActionGroup will be replaced with custom solution and it will be placed outside of the formComponent. It will also solve the issue you mentioned below https://github.com/project-ncl/pnc-web-ui-react/pull/401#discussion_r1499126478